alot/settings/manager.py

```diff
@@ -106,17 +108,26 @@ def read_config(self):
         logging.debug(themes_dir)
 
         # if config contains theme string use that
+        data_dirs = [os.path.join(d, 'alot/themes') for d in DATA_DIRS]
         if themestring:
-            if not os.path.isdir(themes_dir):
-                err_msg = 'cannot find theme %s: themes_dir %s is missing'
-                raise ConfigError(err_msg % (themestring, themes_dir))
+            for dir_ in itertools.chain([themes_dir], data_dirs):
+                if not os.path.isdir(dir_):
+                    logging.warning(
+                        'cannot find theme %s: themes_dir %s is missing',
+                        themestring, dir_)
+                else:
+                    theme_path = os.path.join(dir_, themestring)
+                    try:
+                        self._theme = Theme(theme_path)
+                    except ConfigError as e:
```

Comment:
exactly. for/else is one of my favorite weird python constructs for just that reason.